README.md

```diff
@@ -1 +1,59 @@
-# Catalog-of-my-things
\ No newline at end of file
+# Catalog-of-my-things
+
+In this project, we create a console app that will help you to keep a record of different types of things you won: 
+Books, 
+Music albums, 
+Movies 
+Everything will be based on the UML class diagram presented. The data will be stored in JSON files but we will also prepare a database with tables structure analogical to your program's class structure.
+
+## Built With
+
+- Ruby
+- Postgres
+## Getting Started
+
+- git@github.com:aliabbani/Catalog-of-my-things.git
+
+- cd Catalog-of-my-things
+
```

Comment:
- [ ] I would recommend mentioning the steps or commands to run the project locally. Since this app can only run locally you have to provide clear steps so that any recruiter or user can easily run the application without proper technical knowledge.

Ex :
```
FIrst clone the project with the command 

$ git clone git@github.com:aliabbani/Catalog-of-my-things.git

then go to the right folder 

$ cd Catalog-of-my-things

then run the command

$ ruby main.rb
```